pkg/fleetcrt/fleetcrt.go

```diff
@@ -0,0 +1,91 @@
+// Package fleetcrt contains utilities to load and process TLS certificate files.
+package fleetcrt
```

Comment:
We already have a `pkg/certificate` package with some certificate/x509 helpers, I think that could fit nicely in there.

Reply:
Moved! 935894089.